codegen/src/test/scala/caliban/codegen/SchemaWriterSpec.scala

```diff
@@ -293,6 +293,52 @@ object Types {
   case class Character(name: String)
   case class CharacterArgs(name: String)
 
+}
+"""
+          )
+        )
+      },
+      testM("schema with multiline comment") {
+        val schema =
+          """
+             input CharacterArgs {
+               \"\"\"
```

Comment:
They don't get escaped that way. I am not sure it's doable to escape triples quotes inside triple quotes. Maybe you have to use regular double quotes outside. Can you try?